cyences_app_for_splunk/appserver/static/cs_counter_measure_sophos.js

```diff
@@ -0,0 +1,214 @@
+require([
+  'splunkjs/mvc/tableview',
+  'splunkjs/mvc/chartview',
+  'splunkjs/mvc/searchmanager',
+  'splunkjs/mvc',
+  'underscore',
+  "splunkjs/mvc/utils",
+  'models/SplunkDBase',
```

Comment:
We can remove those which we are not using.

 'models/SplunkDBase',
  'splunkjs/mvc/sharedmodels',
  "splunkjs/mvc/tokenutils",
  "jquery"